riscv/mmu.h

```diff
@@ -71,9 +73,34 @@ class mmu_t
   std::map<reg_t, reg_t> alloc_cache;
   std::vector<std::pair<reg_t, reg_t >> addr_tbl;
 
+  reg_t get_pmlen(bool effective_virt, reg_t effective_priv, xlate_flags_t flags) const {
```

Comment:
How does this handle the case of `MPRV && MXR`?

Reply:
Thanks for the catch. I overlooked the condition.
![image](https://github.com/user-attachments/assets/db12dca2-a2d5-4e16-a70e-2d0a3bcca1ac)

Reply:
Commit https://github.com/riscv-software-src/riscv-isa-sim/pull/1718/commits/95b65c6c5e59af2b204a374410c332eeae7d0510 addresses the ```MPRV && MXR``` condition.